source/browseMode.py

```diff
@@ -1778,7 +1795,10 @@ def doResult(result):
 			obj = container
 		return doResult(False)
 
-	def _get_documentConstantIdentifier(self):
+	documentConstantIdentifier: Optional[str]
+	# Mark documentConstantIdentifier property   for caching during the current core cycle
+	_cache_documentConstantIdentifer = True

```

Comment:
I think doc strings like this are supposed to be a string after the property. I'd also refer to "auto-property" for consistency with the typing doc strings.
```suggestion
	_cache_documentConstantIdentifer = True
	""" Cache auto-property: documentConstantIdentifier during a core cycle. """
```